furusiya/view/renderer.py

```diff
@@ -0,0 +1,149 @@
+import tdl
+
+import colors
+import config
+from constants import FOV_ALGO, FOV_LIGHT_WALLS, MAP_HEIGHT, MAP_WIDTH, color_dark_wall, color_dark_ground, \
+    color_light_wall, color_light_ground, MSG_X, PANEL_Y, SCREEN_WIDTH, PANEL_HEIGHT
+from main_interface import Game
+from model.components.fighter import Fighter
+
+
+def render_all():
```

Comment:
I seriously applaud your honestly heroic efforts at extracting a renderer out of this spaghetti.